wdl2cwl/main_oop.py

```diff
@@ -337,7 +354,7 @@ def main() -> None:
     #     with open(args.output, "w") as result:
     # result.write(str(Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bcftools_stats.wdl"))) #missing args.workflow)
 
-    Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bowtie_1.wdl")
+    Converter.load_wdl_tree("wdl2cwl/tests/wdl_files/bcftools_stats.wdl")
```

Comment:
@Th3nn3ss maybe we can leave both calls above (it's fine if you add it back before we review&merge this PR). Once we get these 2 workflows being translated correctly, then we can start adding more workflows making sure we didn't break others.

Later this will be replaced when we switch the old translator to this one, and use it in the unit tests :+1:

Reply:
both file path will be added to test_miniwdl.py at the end of this PR. Pytest will be conducted on both files.

Reply:
Excellent @Th3nn3ss ! Thanks